extensions/notebook/src/jupyter/jupyterSessionManager.ts

```diff
@@ -170,7 +170,7 @@ export class JupyterSession implements nb.ISession {
 	private _kernel: nb.IKernel;
 	private _messagesComplete: Deferred<void> = new Deferred<void>();
 
-	constructor(private sessionImpl: Session.ISession, skipSettingEnvironmentVars?: boolean) {
+	constructor(private sessionImpl: Session.ISession, skipSettingEnvironmentVars?: boolean, private _pythonEnvVarPath?: string) {
```

Comment:
In the future, is it likely we will want to make this more generic and support additional environment vars? Or is this a special case?

Reply:
This is a special case. But I like the idea of making this extensible.